x/staking/querier/querier.go

```diff
@@ -129,11 +129,10 @@ func NewQueryRedelegationParams(delegatorAddr sdk.AccAddress, srcValidatorAddr s
 }
 
 func queryValidators(ctx sdk.Context, cdc *codec.Codec, k keep.Keeper) (res []byte, err sdk.Error) {
-	stakingParams := k.GetParams(ctx)
-	validators := k.GetValidators(ctx, stakingParams.MaxValidators)
+	validators := k.GetAllValidators(ctx)
```

Comment:
If the query's documentation states _all_ validators instead of _all bonded_ validators, then this change make sense. Is this the expectation @rigelrozanski?